src/api/status/public/js/build-log/check-build-status.js

```diff
@@ -2,7 +2,7 @@
 // being loaded. On staging and production, remove the `api.` subdomain.
 // On localhost:1111, use localhost (drop the port).
 const autodeploymentUrl = (path) =>
-  `//${window.location.hostname.replace('.api', '')}/deploy${path}`;
+  `//${window.location.hostname.replace(/(?:api\.|\.api)/gm, '')}/deploy${path}`;
```

Comment:
I think this can be simplified to `/(api\.|dev\.api\.)/` so it gets rid of the `api` or `dev.api` bits

Reply:
Yes, you're right, so it could be simplified to `/api\./`, right?

Reply:
@humphd Follow what I read [here](https://www.ocpsoft.org/tutorials/regular-expressions/or-in-regex/) 

> If you don’t want your grouping and alternation to interfere with other numbered groups in your expression, each “or” group must be prefixed with `?:`

I guess I try to avoid any cache if not needed, or do you think `/api\./` should be enough without any performance issue ?

Reply:
There's zero performance issue for a regex on a string of this size.